wiki/en/Running-a-Server.md

```diff
@@ -216,15 +164,6 @@ function loadserverstat ( $statfilename )
 ~~~
 
 
-# Servers on the desktop
-
-Jamulus can be configured to run in Server mode on the desktop. This gives you a graphical user interface to control most of the settings. 
-
-* **Windows users** - Use the "Jamulus Server" icon in the Windows Start menu.
-
-* **macOS users** - Double-click the "Jamulus Server" icon in Applications (assuming you put the files from the install there as per [these instructions](Installation-for-Macintosh)).
-
-* **Linux users** - Launch the "Jamulus Server" shortcut. Or you can open a terminal window (`CTRL+ALT+t` on Debian and related distros), type `jamulus -s` and hit return. _(To run a headless Server [read this guide](#installing-and-running-a-server))_
 
 
 ## Server Setup
```

Comment:
Each of the following Server Setup options should show _both_ how to do it from the UI and how to do it from the command line.

Reply:
Line 169 (new) is refering to the _tab name_ in the UI, of course, as is line 191 (new), so these headings need addressing.

Reply:
@gilgongo I think that's worth another PR/issue.